packages/components/card/src/Card.tsx

```diff
@@ -37,8 +38,21 @@ export default defineComponent({
       return children.value.some(node => node.type && (node.type as any).name === 'IxCardGrid')
     })
 
+    const [selected, setSelected] = useControlledProp(props, 'selected', () => false)
+    const isDisabled = computed(() => props.disabled ?? false)
+    const isSelectable = computed(() => props.selectable ?? false)
```

Comment:
```suggestion
    const isDisabled = computed(() => props.disabled)
    const isSelectable = computed(() => props.selectable)
```
因为这两个都有默认值，所以 ?? 没意义。

Reply:
fixed